### ISSUE TITLE
How to use IDataErrorInfo on CollectionControlDialog 

### ISSUE THREAD

**Issue by hanseung-lim** [xceedsoftware/wpftoolkit#1529]
If I use 『string this[string ...]』, There is shown Argument Exception.
Please let me know if there is any good way.

class Car : IDataErrorInfo{
    ....
    public string this[string name]{
        get {return null;}
    }
}
...
CarList car = new CarList();
Xceed.Wpf.Toolkit.CollectionControlDialog dlg = new Xceed.Wpf.Toolkit.CollectionControlDialog(typeof(Car));
dlg.ItemsSource = car;

**Comment by XceedBoucherS** [comment]
Yes, there is a bug in here. Indexer of type string are not correctly managed. The code in CollectionControlDialog.Clone() will need to be reviewed. 

In the meantime, you could always add a try-catch (and do nothing in the catch). Using the Cancel button would not result in a complete clone, but at least the error won't show up. Or do not use a string indexer.

**Comment by hanseung-lim** [comment]
Thank you for your answers.
I'm usong the previous version^^

**Comment by hanseung-lim** [comment]
Thank you for your answers.
Now, I'm using 2.x version^^;
